projects/core/components/primitive-textfield/value-decoration/value-decoration.style.less

```diff
@@ -1,14 +1,23 @@
 :host {
     position: absolute;
     display: block;
-    height: 1.25rem;
-    line-height: 1;
+    height: 2.5rem;
+    line-height: 2.25;
     width: 100%;
-    margin: -1.25rem 0;
-    padding: 0.625rem 0;
+    margin: -1.15rem 0;
     box-sizing: content-box;
-    color: var(--tui-text-01);
+    color: var(--tui-text-03); // @note: override color of text ellipsis
     animation: tuiPresent 1s;
+    vertical-align: middle;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+
+    &:after {
```

Comment:
@waterplea without it, it doesn't align to the baseline

Reply:
Got it